pkg/upgrade/1.0.0-1.1.0/v1alpha1/csp_upgrade.go

```diff
@@ -113,7 +114,7 @@ func patchCSP(cspObj *apis.CStorPool) error {
 	cspVersion := cspObj.Labels["openebs.io/version"]
 	if cspVersion == currentVersion {
 		tmpl, err := template.New("cspPatch").
-			Parse(templates.OpenebsVersionPatch)
+			Parse(templates.VersionDetailsPatch)
```

Comment:
`patch` is being done later in the code. Will that stay or will that be changed to `update`?

Reply:
Patch will stay.